2.6.2 Changes

IceDragon200 opened the pull request:
## Builds

**2016-07-14 @ 22:13:22 EST** 358abea [2.6.1 - Download](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.6.1/growthcraft-1.7.10-2.6.1-complete--358abea.jar)

> #### Checksums
> 
> **MD5**: 3cd334e0bbf88daa980e925ba11f2fdf
> **SHA1**: e531e90d86c7ade56a522ecd5ae89d22fed0976a
> **SHA256**: 3962c305e3ac0ac49e440c07fa8127a1455db5d57b5922a17d932e61b6a7916b
> #### Changes
> - Disabled NEI handlers indefinitely
## Older Builds

**2016-07-07 @ 09:36:26 EST** bbe16fb [2.6.1 - Download](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.6.1/growthcraft-1.7.10-2.6.1-complete--bbe16fb.jar)

> #### Checksums
> 
> **MD5**: c6056ed504f31e534eafcaa515bbb17c
> **SHA1**: 7d3a28b30e2bf6b270bb794ede37e91daf1f6615
> **SHA256**: a1775bc87520931c058e41debd0267dbe96e7326dc975a899e2eb741798f176a
> #### Changes
> - Replaced Thistle Seed texture with one from Mangoose on the forums

**2016-07-05 @ 10:46:23 EST** d348b38 [2.6.1 - Download](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.6.1/growthcraft-1.7.10-2.6.1-complete--d348b38.jar)

> #### Checksums
> 
> **MD5**: ac3ddd88c4d87c823e3453ff3a84829a
> **SHA1**: a120a3809b89a9bb3ccd2b88ea71c16bf73c2df4
> **SHA256**: 5c1936951231defddcd5cf25724d1ddf37d76dfaab320f923739f4ff7a6a4e99
> #### Changes
> - Added Thistle en_US localization (Thanks @Sunconure11)
> - Added Thistle seeds to listAllseed (Thanks @Sunconure11)

**2016-07-04 @ 23:34:20 EST** c8b4946 [2.6.1 - Download](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.6.1/growthcraft-1.7.10-2.6.1-complete--c8b4946.jar)

> #### Checksums
> 
> **MD5**: c84da5940c24a7699079352fbbd693e3
> **SHA1**: eb068f6594277fa1f6833bdd8c35b1a8593641b3
> **SHA256**: cd5454e9582be768ec201f5772cccc4f0a168385e2f4a466544b7dd5b01d71fd
> #### Changes
> - Added Thistle Seed Item (untested)
> - Thistle now grows in stages (untested)
> - Thistle will spawn as seedlings (untested)
> - Thistle is evil (tested and known.)

**2016-07-01 @ 19:59:44 EST** 43b51b0 [2.6.1 - Download](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.6.1/growthcraft-1.7.10-2.6.1-complete--43b51b0.jar)

> #### Checksums
> 
> **MD5**: 41d39499baf066dc87ea5bef34861baa
> **SHA1**: 1a15c11f969e1b4d7bc3e862b624fb1bcdf24a95
> **SHA256**: 5ebcc6135787ce4fd88a38c3b99de1da541f38a335df5a178685a2ccfbe13909
> #### Changes
> - Thistle can now be toggled via config option
> - Thistle worldgen can now be toggled via config option
> - More logging level changes (moved several `info` level logs to `debug`)

**2016-06-30 @ 10:44:01 EST** 323d762 [2.6.1 - Download](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.6.1/growthcraft-1.7.10-2.6.1-complete--323d762.jar)

> #### Checksums
> 
> **MD5**: a8c0717a1b01d5d756a5def17031ba7e
> **SHA1**: a85e9dc61d4983a5ce1fa5c4409838716d87ba1b
> **SHA256**: a19861642396289c377ce5e1373bef567726fe720715d691e8f2352e00f62f55
> #### Changes
> - Removed Thistle world gen logging
> - Shifted some logs up from INFO to DEBUG

**2016-06-26 @ 08:38:54 EST** 02791e3 [2.6.1 - Download](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.6.1/growthcraft-1.7.10-2.6.1-complete--02791e3.jar)

> #### Checksums
> 
> **MD5**: af6ce1342a58fc5abe7216d69a35b573
> **SHA1**: 64d80e8aac83f37f5e6320218046e3dc5b323083
> **SHA256**: 40ff6236719e31c6001a5dd70b37b9b284f83b1386d4ab2a2adac78d9ba792a8
> #### Changes
> - Reduce Thistle Spread Range (it is now only 1 block in each direction)

**2016-06-21 @ 09:53:34 EST** 0910dd6 [2.6.1 - Download](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.6.1/growthcraft-1.7.10-2.6.1-complete--0910dd6.jar)

> #### Checksums
> 
> **MD5**: 179a4a022d88ea02825b3e202c8747df
> **SHA1**: 3b927edaa606be02e847870fc60259074391b934
> **SHA256**: 87d4a3079fbb11827eae5eadcb880cb54a95f13485aee5464d7d722d6c2ad502
> #### Changes
> - Added new config option for controlling thistle world gen spawn rate (on worldgen)

**2016-06-18 @ 18:15:01 EST** 3f37aac [2.6.1 - Download](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.6.1/growthcraft-1.7.10-2.6.1-complete--3f37aac.jar)

> #### Checksums
> 
> **MD5**: 0884a0f77016872ac60947130bf385a5
> **SHA1**: e47872fe2fbe7fabd73fcd4adeabe35cee264114
> **SHA256**: 672ac4217b39ae69707724ace19b01c506ba602cef168987c70759901e87d85c
> #### Changes
> - Fixed Cellar tiles crashing on gui network sync
## Fixes
- #385 Fixed Cellar tiles crashing during a gui sync

IceDragon200 committed:
cellar: Fixed TileEntity GuiNetworkSync crashing when the id was out of range

Alatyami commented:
Planning on releasing on Friday if there are no other significant bugs by then.

IceDragon200 committed:
milk: Added new config option to control thistle world gen

IceDragon200 committed:
milk: Reduce Thistle spread range

IceDragon200 committed:
cellar,milk: Get logging under control

For the most part >:

Fixes #392

IceDragon200 committed:
milk: Thistle block, and or world gen can now be disabled

IceDragon200 committed:
core: ConfigBase config option logging should be done at debug level

IceDragon200 committed:
api,core,cellar: Shifted a few more log levels to DEBUG

Alatyami committed:
Added texture for Thistle Seeds

Sunconure11 committed:
milk: Thistle stages [textures] (#394)

IceDragon200 committed:
milk: Setting thistle spread chance to 0 will disable it

IceDragon200 committed:
milk: Implemented Thistle and Thistle seed

Though I think I'm forgetting the grass hook, and also the recipe...

IceDragon200 committed:
milk: Prevent Thistle Seed from being created and registered if thistle was disabled in config

Also added grass hook with customizable seed weight

IceDragon200 committed:
milk: Allows users to disable thistle seeds

IceDragon200 committed:
milk: Breaking a thistle flower will give different items based on it's growth stage

A fully grown thistle will drop itself, otherwise it should drop it's seeds, if available, otherwise NOTHING is dropped.

Sunconure11 committed:
milk: en_US; Added localizations for thistle seeds (#396)

Sunconure11 committed:
milk: Added oredictionary definitions for thistle seeds (#395)

* Added oredictionary definitions for seeds

* Update GrcMilkItems.java

IceDragon200 committed:
assets/all*: Refreshed lang files

IceDragon200 committed:
Added ATTRIBUTION.json

Now we can keep track of who made what

IceDragon200 committed:
attribution: Missed the black crowbar

IceDragon200 committed:
attribution: IceDragon, lord of the typos

Sunconure11 committed:
milk: Mangoose's thistle seed textures (#397)

Credits go to Mangoose of the Minecraft Forums

IceDragon200 committed:
attribution: Thistle seed was replaced with one made by mangoose

IceDragon200 committed:
milk: !! Disable Thistle World Gen by default

To any user reading this, be sure to disable Thistle, unless you like the weedy nature of it

cadaverous-eris commented:
Is this going to be released?

IceDragon200 commented:
@frederick-the-nameless It is planned. 
Since the NEI is STILL broken, it will be removed in 2.6.2 until further notice.

IceDragon200 committed:
cellar,core: Disable NEI handlers

Also removed options for toggling NEI support.

This disabling will be indefinite

IceDragon200 committed:
milk: Explicit variable initialization to false should be avoided

Or maybe I should enable it... dunno

Commit:
Update gradle.properties
